lib/jacoco.gradle

```diff
@@ -0,0 +1,75 @@
+apply plugin: 'jacoco'
+
+jacoco {
+    // Need this specific version due to https://stackoverflow.com/questions/59378618/kotlin-jacoco-no-coverage-illegalclassformatexception-please-supply-orig
+    toolVersion = "0.7.9"
```

Comment:
Ouch. I created https://github.com/relaycorp/relaynet-endpoint-android/issues/6. Hopefully the fix will be released soon, because I've had issues with coroutines in older versions of JaCoCo.